tests/modules/datasets/tasks/test_s3_access_point_share_manager.py

```diff
@@ -1347,18 +1339,20 @@ def test_delete_dataset_bucket_key_policy_existing_policy_with_no_additional_tar
     # Given
     kms_client = mock_kms_client(mocker)
     kms_client().get_key_id.return_value = "1"
+    target_requester_arn = f"arn:aws:iam::{TARGET_ACCOUNT_ENV}:role/{TARGET_ACCOUNT_ENV_ROLE_NAME}"
```

Comment:
very minor, we can use the manager to replace this line right? like `manager.get_role_arn(TARGET_ACCOUNT_ENV, TARGET_ACCOUNT_ENV_ROLE_NAME)` This way we also test that small method

Reply:
Addressed this in next revision.